docs/advanced-usage.md

```diff
@@ -24,6 +24,8 @@ These affect ORCA in all contexts.
 
 * <a name="ORCA_PHPCS_STANDARD"></a>**`ORCA_PHPCS_STANDARD`**: Change the PHP Code Sniffer standard used by the `qa:static-analysis` and `qa:fixer` commands. Acceptable values are `AcquiaPHP`, `AcquiaDrupalStrict`, and `AcquiaDrupalTransitional`. See [Acquia Coding Standards for PHP](https://packagist.org/packages/acquia/coding-standards) for details.
 
+* <a name="ORCA_SUT_DIR"></a>**`ORCA_SUT_DIR`**: Change the path which ORCA uses as a SUT. Accepted values are any valid local directory reference, e.g., `/var/www/example`, or `../example`.
```

Comment:
Good job noticing the pattern for directory examples. Just a couple of tweaks for precision:

```suggestion
* <a name="ORCA_SUT_DIR"></a>**`ORCA_SUT_DIR`**: Change the path where ORCA looks for the SUT. Accepted values are any valid directory path, e.g., `/var/www/example`, or `../example`.
```